api-extractor.json

```diff
@@ -10,7 +10,8 @@
   "dtsRollup": {
     "enabled": true,
     "untrimmedFilePath": "",
-    "publicTrimmedFilePath": "<projectFolder>/<unscopedPackageName>.d.ts"
+    "publicTrimmedFilePath": "<projectFolder>/<unscopedPackageName>.d.ts",
+    "betaTrimmedFilePath": "<projectFolder>/lib/beta.d.ts"
```

Comment:
Are there any other features currently marked experimental that we want to expose via the beta tag? Have we talked to Alex about how we are going to differentiate messaging in experimental-tagged vs beta-tagged features? (e.g., we currently have a "use at your own risk" disclaimer for experimental features, which is missing from the release highlight for this feature - I imagine we want to add something about the beta aspects). Similarly, how will users know about the existence of the beta namespace and how to access it?

Reply:
> Are there any other features currently marked experimental that we want to expose via the beta tag?

No.

> Have we talked to Alex about how we are going to differentiate messaging in experimental-tagged vs beta-tagged features?

No, but I can ask.  There's also no reason we can't tag everything that is tagged beta as experimental - beta is only used to indicate to API extractor what should be extracted into the beta `.d.ts` file.

> Similarly, how will users know about the existence of the beta namespace and how to access it?

I don't think the intention is for users to "find" the beta namespace and explore it.  Instead, users will find a beta feature, either through release notes or documentation or a blog post if Alex wants to write one, and our reference will point users towards the beta namespace when showing them how to use the feature.  There's already a docsp ticket filed for this work.

Reply:
Do you mean that we could label the same thing as both beta and experimental and it's the experimental tag that indicates non-stability? Could you clarify what the beta tag should signify to users?

Reply:
Yes.  That's also what is currently implemented in this PR.

The beta tag, if users see it, indicates the same thing.  But it's only present because API extractor requires it to extract the beta API.

Reply:
(I assume it's just for actually surfacing the feature in the docs and would always be paired with experimental, but I wanted to make sure - it's something that we'll want to capture either in our internal team FAQ or in this repo's readme for future reference)

Reply:
Thanks - I've now added a note to our internal FAQ on pre-release features about the beta tag. Could you add our standard note on `@experimental` features to the release highlight text? Once that's done, we can resolve this thread.